src/PAModel/PAConvert/Yaml/YamlWriter.cs

```diff
@@ -149,5 +145,10 @@ private void WriteIndent()
                 _text.Write(Indent);
             }
         }
+
+        public static string NormalizeNewlines(string x)
```

Comment:
>NormalizeNewlines [](http://example.com/codeflow?start=29&length=17)

Can we add some more dedicated tests for this?

There are so many corner cases here. 

\r, \n, \r\n --> single
\r\r,  \n\n --> double